pyanaconda/payload/dnfpayload.py

```diff
@@ -412,80 +414,149 @@ def addRepo(self, ksrepo):
         super(DNFPayload, self).addRepo(ksrepo)
 
     def _apply_selections(self):
+        log.debug("applying DNF package/group/module selection")
+
+        # enable modules
+        for module in self.data.module.dataList():
+            # stream definition is optional
+            if module.stream:
+                module_spec = "{name}:{stream}".format(name=module.name, stream=module.stream)
+            else:
+                module_spec = module.name
+            log.debug("enabling module: {}".format(module_spec))
+            # TODO:
+            # - is this cleared by self._base.reset() ?
+            # - what does the save_immediately parameter of enable_module() do ?
+            self._base.enable_module([module_spec])
+
+        # handle "normal" groups
+        group_specs = set()
```

Comment:
I would prefer `selected_group_specs`, so it is consistent with other variables.

Reply:
Dropped in favor of common include/exclude sets.